BICCN-policies/dataset-metadata.yaml

```diff
@@ -0,0 +1,85 @@
+identifier: REQUIRED ## Post upload (or during dandi organize)
+name: REQUIRED
+description: REQUIRED
+contributors: # required for author and contact
+- orcid: # REQUIRED
+  roles: # REQUIRED from https://casrai.org/credit/ + maintainer, contact
+  email: Recommended # filled from orcid
+  name: Recommended  # filled from orcid
+  affiliations: optional  # filled from orcid
+sponsors:
+- identifier: RECOMMENDED
+  name: REQUIRED
+  url: RECOMMENDED
+license:
+- url # REQUIRED
+keywords: [key1, key2,]
+consortium/project:
+- name: REQUIRED
+  identifier: REQUIRED #RRID
+associated_disease: # RECOMMENDED
+- name: REQUIRED
+  identifier: REQUIRED
+associated_anatomy: # RECOMMENDED
+- name: REQUIRED
+  identifier: REQUIRED
+protocols: # OPTIONAL
+- name: REQUIRED
+  identifier: REQUIRED
+ethicsApprovals: RECOMMENDED
+- name: REQUIRED # name of committee
+  country: REQUIRED
+  identifier: REQUIRED # protocol number
+access: OPTIONAL # 
+  status: REQUIRED # open, embargoed, restricted
+  access_request_url: REQUIRED for embargoed and restricted
+  access_contact_email: REQUIRED for embargoed and restricted
+language: RECOMMENDED
+
+## All metadata below could either be added after publication or
+## extracted from NWB files
+
+## Post publication (i.e. added by DANDI)
+version: REQUIRED
+releaseDate: REQUIRED
+associatedData:
```

Comment:
Associated data is set up here to refer to the raw data.  The associated data has a repository and url.
During the collection of dataset metadata from BICCN labs, several labs provided me with an alternate data access URL like brainome.org. Where should we model this alternate/related data access URL?